x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/planner/QueryFolder.java

```diff
@@ -19,6 +26,60 @@ PhysicalPlan fold(PhysicalPlan plan) {
 
     @Override
     protected Iterable<RuleExecutor<PhysicalPlan>.Batch> batches() {
+        Batch fold = new Batch("Fold queries",
+                new FoldFilter()
+        );
+        Batch finish = new Batch("Finish query", Limiter.ONCE,
+                new PlanOutputToQueryRef()
+        );
+        
+        //return Arrays.asList(fold, finish);
```

Comment:
I'm assuming this is intentional and will be uncommented out later?

Reply:
Yes.